orsopy/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Top-level package for orsopy."""
 
 __author__ = """Andrew R. McCluskey"""
-__email__ = 'andrew.mccluskey@ess.eu'
-__version__ = '0.0.3'
+__email__ = "andrew.mccluskey@ess.eu"
```

Comment:
We should maybe remove this since it is becoming more and more redundant with more authors

Reply:
Agreed, better to keep this on github/documentation files than the package directly.
I checked that the setup does not use these variables.